counterexamples/map_floor.lean

```diff
@@ -119,7 +119,8 @@ itself. -/
 lemma lt_forget_epsilons_ceil (n : ℤ) :
   ⌈forget_epsilons (n + ε)⌉ < forget_epsilons ⌈(n + ε : ℤ[ε])⌉ :=
 begin
-  rw [←neg_lt_neg_iff, ←map_neg, ←cast_neg, ←floor_neg, ←floor_neg, ←map_neg, neg_add', ←cast_neg],
+  rw [←neg_lt_neg_iff, ←map_neg, ←cast_neg, ←floor_neg_eq_neg_ceil, ←floor_neg_eq_neg_ceil,
+  ←map_neg, neg_add', ←cast_neg],
```

Comment:
Why did that change actually?

Reply:
Oh, I added a lemma that really wanted to be called floor_neg to match a pattern.

Reply:
Can you rename your `floor_neg`/`floor_neg'` to `floor_neg_iff` and revert that name change?

Reply:
Will do.